lib/esextractor.cpp

```diff
@@ -133,6 +140,12 @@ es_extractor_video_codec (ESExtractor * extractor)
   return extractor->codec ();
 }
 
+ESEVideoFormat
+es_extractor_video_format (ESExtractor * extractor)
```

Comment:
Always a bit nervous with raw pointers and no null checks, but it looks like this invariant is pretty strongly maintained. Would be nice if `nonnull` attributes were standard.

Reply:
yes I should test it everywhere